master/internal/rm/kubernetesrm/resource_pool_intg_test.go

```diff
@@ -1044,31 +1126,6 @@ func TestValidateResources(t *testing.T) {
 	}
 }
 
-func TestSchedule(t *testing.T) {
```

Comment:
I removed this test because what it tries to cover is duplicated by other tests.